sql/updates/reverse-dev.sql

```diff
@@ -31,3 +31,119 @@ CREATE FUNCTION @extschema@.add_continuous_aggregate_policy(
 RETURNS INTEGER
 AS '@MODULE_PATHNAME@', 'ts_update_placeholder'
 LANGUAGE C VOLATILE;
+
+DROP PROCEDURE
+_timescaledb_functions.policy_compression_execute(
+  job_id              INTEGER,
+  htid                INTEGER,
+  lag                 ANYELEMENT,
+  maxchunks           INTEGER,
+  verbose_log         BOOLEAN,
+  recompress_enabled  BOOLEAN,
+  use_creation_time   BOOLEAN,
+  useam               BOOLEAN = NULL);
+
+CREATE PROCEDURE
```

Comment:
```suggestion
CREATE OR REPLACE PROCEDURE
```

Reply:
we just dropped this in the previous line. so why use create or replace?